content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -16,35 +16,49 @@ apply changes to a cluster.
 
 You can analyze your current live Kubernetes cluster by running:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=analyze_all_namespaces >}}
 $ istioctl analyze --all-namespaces
 {{< /text >}}
 
 And that’s it! It’ll give you any recommendations that apply.
 
 For example, if you forgot to enable Istio injection (a very common issue), you would get the following warning:
 
-{{< text plain >}}
-Warn [IST0102](Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
+{{< text syntax=plain snip_id=analyze_all_namespace_sample_response >}}
+Warn [IST0102] (Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
+{{< /text >}}
+
+Fix the issue:
+
+{{< text syntax=bash snip_id=fix_default_namespace >}}
+$ kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+Then try again:
+
+{{< text syntax=bash snip_id=try_with_fixed_namespace >}}
+$ istioctl analyze --namespace default
+✔ No validation issues found when analyzing namespace: default.
 {{< /text >}}
 
 ## Analyzing live clusters, local files, or both
 
 Analyze the current live cluster, simulating the effect of applying additional yaml files like `bookinfo-gateway.yaml` and `destination-rule-all.yaml` in the `samples/bookinfo/networking` directory:
 
-{{< text bash >}}
-$ istioctl analyze @samples/bookinfo/networking/bookinfo-gateway.yaml@ @samples/bookinfo/networking/destination-rule-all.yaml@
+{{< text syntax=bash snip_id=analyze_sample_destrule >}}
+$ istioctl analyze samples/bookinfo/networking/bookinfo-gateway.yaml samples/bookinfo/networking/destination-rule-all.yaml
```

Comment:
YAML files were clickable links.

```suggestion
$ istioctl analyze @samples/bookinfo/networking/bookinfo-gateway.yaml@ @samples/bookinfo/networking/destination-rule-all.yaml@
```

Reply:
@frankbu They were!  I would like to bring that back.  I went through the _.md_ files and couldn't find any instances of `@samples` in `< text snip_id=... >` blocks.  Can you offer style guidance for this situation?

Reply:
Most docs containing commands that use yaml files in the github samples dir wrap them in `@` chars so that the file is clickable in the UI. That includes this command (Look at line 36 in the diff, above, which has the `@` chars that you removed). https://preliminary.istio.io/latest/docs/ops/diagnostic-tools/istioctl-analyze/#analyzing-live-clusters-local-files-or-both. I'm just saying leave it as it was, don't change this in the doc.